### PR TITLE
Static Web Content

### PR DESCRIPTION
**Pull request by nzoschke**

### CODE REVIEW

File: docs/static-sites.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+
+Compare this to a traditional Model View Controller (MVC) approach like Rails or Django. In this architecture the API may spend lots of time rendering HTML, and the HTML may not get served to users if there is an application bug or a database outage.
+
+Static websites are a solved problem on AWS. We simply create an S3 bucket configured for website hosting and upload the content with public-read permissions. Then anyone can access the content from a URL like `http://gofaas-webbucket-572007530218.s3-website-us-east-1.amazonaws.com` with some of the highest reliability and lowest storage and bandwidth costs possible.
+
+Serving this from a custom domain is also a solved problem. We add the CloudFront CDN, configured with an SSL cert via the AWS Certificate Manager, in front of the S3 bucket. When we point our custom domain DNS to CloudFront, users can access the content from a URL like `https://www.gofaas.net` with some of the fastest delivery times and lowest bandwidth costs possible thanks to the global content caching network.
```
Comment (mwarkentin) [inline docs/static-sites.md]: Need any information about invalidating cached objects during deploys?

Comment (nzoschke) [reply docs/static-sites.md]: Good call. Also missing in the `make deploy-static` command

**Comment by nzoschke** [comment]
This should mention something about ACM email verification too.

**Comment by mwarkentin** [comment]
ACM supports DNS-based verification now - not sure if that could be worked into route53 for full automation: https://docs.aws.amazon.com/acm/latest/userguide/gs-acm-validate-dns.html

**Pull request merged**